doc/man7/crypto.pod

```diff
@@ -210,6 +211,60 @@ property string from the B<EVP_PKEY_CTX>.
 
 =back
 
+=head2 Performance
+
+If you perform the same operation many times then it is recommeded to use
+L</Explicit fetching>. An algorithm may be prefetched once initially,
+and then passed to any operations that are currently using L</Implicit fetching>.
+See an example of Explicit fecthing in L</USING ALGORITHMS IN APPLICATIONS>.
+
+If the prefetched object is not passed to operations, then any implicit
+fetch will use the internally cached prefetched object, but it will
+still be slower than passing the prefetched object directly.
+
+Prior to OpenSSL 3.0, constant method tables (such as EVP_sha256()) were used
```

Comment:
Something should be mentioned about these functions changing.  I.e. it's not clear that they aren't returning constant method tables in 3.x from the text here (sure, the implicit fetch above kind of says this, but making it clear in the performance section seems reasonable).

Reply:
Please suggest some text? I am not sure how that will make things clearer.

Reply:
I have attempted to update it, see if that helps..